types/simulation/types.go

```diff
@@ -173,9 +173,8 @@ type AppStateFn func(r *rand.Rand, accs []Account, config Config) (
 )
 
 // AppStateFnWithExtendedCb returns the app state json bytes and the genesis accounts
-type AppStateFnWithExtendedCb func(r *rand.Rand, accs []Account, config Config) (
-	appState json.RawMessage, accounts []Account, chainId string, genesisTimestamp time.Time,
-)
+// Deprecated: Use AppStateFn instead. This will be removed in a future relase.
+type AppStateFnWithExtendedCb AppStateFn
```

Comment:
Even if not used, this makes v0.47 still not API breaking: https://github.com/cosmos/cosmos-sdk/pull/15903#issuecomment-1518452739.
It will be then deleted in v0.48.